parquet-format-structures/src/main/java/org/apache/parquet/format/LogicalTypes.java

```diff
@@ -53,4 +53,5 @@ public static LogicalType DECIMAL(int scale, int precision) {
   public static final LogicalType JSON = LogicalType.JSON(new JsonType());
   public static final LogicalType BSON = LogicalType.BSON(new BsonType());
   public static final LogicalType FLOAT16 = LogicalType.FLOAT16(new Float16Type());
+  public static final LogicalType VARIANT = LogicalType.VARIANT(new VariantType());
```

Comment:
Didn't we add a version number to the format? I think that we need to write version=1 and verify that the version is 1 when reading.

Reply:
Yeah. We added the version.